### PR TITLE
Update IaC documentation

### PR DESCRIPTION
**Pull request by fsismondi**
Some suggestions for the IaC documentation section

### CODE REVIEW

File: docs/modules/ROOT/pages/explanations/infrastructure-as-code.adoc [hunk 1]
```diff
@@ -1,21 +1,23 @@
...
+= Infrastructure as Code (IaC)
 
 Deploying Kubernetes clusters from scratch has become an easy task with IaaS interfaces.
 However, the proper management of all components that make up the cluster
 and its ecosystem can quickly become a nightmare if everything was done manually.
 
-The Infrastructure as Code approach has long proved its worth to deploy
-and maintain complex environments on the long run.
-In particular, in the context of Cloud resources, it allows to make sure
+The IaC approach has long proved its worth when deploying and maintaining complex environments
```
Comment (raphink) [inline docs/modules/ROOT/pages/explanations/infrastructure-as-code.adoc:7]: I'm not sure about removing "Infrastructure as Code", which I still find clearer.

File: docs/modules/ROOT/pages/explanations/infrastructure-as-code.adoc [hunk 2]
```diff
@@ -1,21 +1,23 @@
...
 Deploying Kubernetes clusters from scratch has become an easy task with IaaS interfaces.
 However, the proper management of all components that make up the cluster
 and its ecosystem can quickly become a nightmare if everything was done manually.
 
-The Infrastructure as Code approach has long proved its worth to deploy
-and maintain complex environments on the long run.
-In particular, in the context of Cloud resources, it allows to make sure
+The IaC approach has long proved its worth when deploying and maintaining complex environments 
+on the long run.
+In particular, in the context of Cloud resources, it allows sysadmins to make sure
```
Comment (raphink) [inline docs/modules/ROOT/pages/explanations/infrastructure-as-code.adoc:9]: Not really. Some devs also use IaC to manage cloud resources of their own (RDS databases for ex). Some of our devs use Terraform actually.

File: docs/modules/ROOT/pages/explanations/infrastructure-as-code.adoc [hunk 3]
```diff
@@ -1,21 +1,23 @@
...
 all resources are managed centrally and destroyed in their proper context.
-This is important to control the financial consequences of numerous deployments.
+These two are extremely important for keeping control over the economical cost of your numerous
```
Comment (raphink) [inline docs/modules/ROOT/pages/explanations/infrastructure-as-code.adoc:11]: Actually, I'd like to go through the whole doc and get rid of personal sentences (such as "you", "your", "we", "camptocamp") to replace them with more neutral expressions.

File: docs/modules/ROOT/pages/explanations/infrastructure-as-code.adoc (hunk 1 above)
Comment (fsismondi) [reply docs/modules/ROOT/pages/explanations/infrastructure-as-code.adoc:7]: isnt IaC a well-known term used the same way IaaS is?
see that now article's title includes what the acronym stands for

File: docs/modules/ROOT/pages/explanations/infrastructure-as-code.adoc (hunk 3 above)
Comment (fsismondi) [reply docs/modules/ROOT/pages/explanations/infrastructure-as-code.adoc:11]: Makes sense. Let's discuss styling and then I can contribute reviewing the whole doc

**Pull request closed without merging**